src/components/FileUploadField/FileUploadField.tsx

```diff
@@ -11,6 +11,13 @@ import TextInput from '../TextInput';
 import TextList from '../TextList';
 import TextListItem from '../TextListItem';
 
+// Remove native properties from TS interface (https://github.com/microsoft/TypeScript/blob/c388222b29228a572b4e3c9ca907863d0b1945c6/lib/lib.dom.d.ts#L2537)
+// that don't need to be passed or saved
+type FileInfo = Omit<
```

Comment:
i'm not 100% sure this is the right thing to do but we're not actively working on this component yet anyways 😅